repos/c2sm/packages/icon/package.py

```diff
@@ -671,15 +673,20 @@ def checksuite(self):
             # test.py fails if PYTHONHOME has any value,
             # even '' or ' ' is failing, therefore delete
             # it temporary from env
-            PYTHONHOME = os.environ['PYTHONHOME']
-            os.environ.pop('PYTHONHOME')
+            try:
+                PYTHONHOME = os.environ['PYTHONHOME']
+                os.environ.pop('PYTHONHOME')
+                pythonhome_is_set = True
+            except:
+                pythonhome_is_set = False
```

Comment:
Consider using `if 'PYTHONHOME' in os.environ:`

Reply:
Done